smart-contracts/contracts-common/concordium-contracts-common/src/schema.rs

```diff
@@ -1020,7 +1020,7 @@ pub fn deserial_length(source: &mut impl Read, size_len: SizeLength) -> ParseRes
 
 // Versioned schema helpers
 #[cfg(feature = "derive-serde")]
-mod impls {
+pub mod impls {
```

Comment:
We don't want this. We don't want to expose the fact that there is an `impls` module publicly.
If specific functions are needed then they should be publicly exported from the library using `pub use`.

Reply:
Have added 
```
#[cfg(feature = "derive-serde")]
pub use impls::VersionedSchemaError;
```

in top of file `schema.rs`. Let me know if it is the wrong place to put it.

Reply:
It's fine.